.git-authors

```diff
@@ -18,3 +19,4 @@ email_addresses:
    kj: karthijey@yahoo.com
    km: klausmana@outlook.com
    mw: matt_wills8@hotmail.co.uk
+   nc: nelsonc19@gmail.com
```

Comment:
- `gmail` is default, so you do not need to add a line here. 
- `nc: Nelson Cothran` should become `nc: Nelson Cothran; nelsonc19`.

Reply:
Yup, as @bbmoz said the default domain setting is `gmail.com`.
You can see it at `.git-authors` file, line 15:
```
email:
  prefix: pair
  **domain: gmail.com**
```
Thank you for good review from @bbmoz and @nel-co do not hesitate to change it.